### PR TITLE
opengl is now buildable again - but doesn't render :(

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

File: synfig-core/configure.ac [hunk 1]
```diff
@@ -347,7 +347,7 @@ AC_ARG_WITH(opengl,
 ],[
 	with_opengl="no"
 ])
-if test $with_opengl != "no" ; then {
+if test $with_opengl = "no" ; then {
```
Comment (ice0) [inline synfig-core/configure.ac:350]: Looks like this will break logic?
It looks for `gl.pc`, only if you say `--with_opengl`

Comment (rodolforg) [reply synfig-core/configure.ac:350]: Actually, no.
https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/External-Software.html

Comment (ice0) [reply synfig-core/configure.ac:350]:
```
./configure --without-opengl
```

```
OpenGL ---------------------------> yes


OpenCL ---------------------------> no
libdv ----------------------------> yes
OpenEXR --------------------------> yes
```

Definitely not the behavior the user expects.

Comment (rodolforg) [reply synfig-core/configure.ac:350]: Well, I believe you :)
This commit was made on May 24 :P I just rebased and pushed now.
Maybe I changed in a bad way to autobuild include opengl support without touch it and forgot to not commit back then.
Well, I'll remove those changes. Thank you for check it (and sorry for that)

Comment (rodolforg) [reply synfig-core/configure.ac]: Done

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by rodolforg** [comment]
Partial fix #1843

**Pull request merged**